.gitignore

```diff
@@ -58,6 +58,7 @@ macos/gobridge/wireguard-go-version.h
 .vscode/
 MozillaVPN.xcodeproj/
 MozillaVPN.vcxproj*
+*.autosave
```

Comment:
Not sure if this change should be in this PR since it is not related.

Reply:
It's a good addition though!